mmcls/datasets/transforms/formatting.py

```diff
@@ -157,11 +157,9 @@ def transform(self, results: dict) -> dict:
                 'in the data pipeline or images have been loaded in ')
 
         data_sample = MultiTaskDataSample(tasks=self.tasks)
```

Comment:
This is strange to put a tasks keyword in the MultiTaskDataSample constructor
The only possible argument is `metainfo=`

Please remove the tasks options in the pipeline, let's make it simple for now